# Catalan translation for update-manager
# Copyright (C) 2006
# This file is distributed under the same license as the update-manager package.
# Jordi Irazuzta Cardús <irazuzta@gmail.com>, 2006.
#
#: ../DistUpgrade/DistUpgradeController.py:1075
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2020-08-03 09:26-0700\n"
"PO-Revision-Date: 2019-06-18 17:52+0000\n"
"Last-Translator: Walter Garcia-Fontes <walter.garcia@upf.edu>\n"
"Language-Team: Catalan <tradgnome@softcatala.org>\n"
"Language: ca\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:46+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "Servidor per %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Servidor principal"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Servidors personalitzats"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "No s'ha pogut calcular l'entrada del fitxer sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"No s'ha pogut trobar cap paquet, esteu utilitzat un disc de l'Ubuntu per a "
"l'arquitectura adequada?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "No s'ha pogut afegir el CD"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"S'ha produït un error en afegir el CD i l'actualització s'ha cancel·lat. "
"Informeu d'aquest error si això ha passat amb un CD d'Ubuntu vàlid.\n"
"\n"
"El missatge d'error ha estat:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Elimina el paquet en mal estat"
msgstr[1] "Elimina els paquets en mal estat"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"El paquet «%s» es troba en un estat inconsistent i s'ha de tornar a "
"instal·lar, però no s'ha trobat l'arxiu per a fer-ho. Voleu eliminar aquest "
"paquet i continuar?"
msgstr[1] ""
"Els paquets «%s» es troben en un estat inconsistent i s'han de tornar a "
"instal·lar, però no s'ha trobat els arxius per a fer-ho. Voleu eliminar "
"aquests paquets i continuar?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "Pot ser que el servidor estigui sobrecarregat"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "Paquets trencats"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"El vostre sistema conté paquets trencats que no es poden arreglar amb "
"aquesta aplicació. Utilitzeu el Synaptic o l'apt-get abans de continuar."

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Paquets trencats"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Això és probablement un problema transitori, torneu a provar-ho més tard."

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "No s'ha pogut calcular l'actualització"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "S'ha produït un error en autenticar alguns paquets"

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"No s'han pogut autenticar alguns paquets. Potser hi ha un problema temporal "
"amb la xarxa. Podeu provar-ho més tard. A continuació es mostra la llista "
"amb els paquets no autenticats."

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"El paquet «%s» està marcat per a eliminar-lo, però apareix a la llista negra "
"de fitxers a eliminar."

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "El paquet essencial «%s» està marcat per a ésser eliminat."

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "S'està intentant instal·lar la versió «%s» de la llista negra"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "No s'ha pogut instal·lar «%s»"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Ha sigut impossible instal·lar el paquet necessari. Informeu d'aquest error "
"utilitzant l'ordre «ubuntu-bug ubuntu-release-upgrader-core» en un terminal."

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "No s'ha pogut conjecturar el metapaquet"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"El vostre sistema no conté un paquet %s o %s i no ha estat possible detectar "
"la versió de l'Ubuntu que esteu fent servir.\n"
" Instal·leu un dels paquets mencionats fent servir el synaptic o l'apt-get "
"abans de continuar."

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "S'està llegint la memòria cau"

#: ../DistUpgrade/DistUpgradeController.py:225
msgid "Unable to get exclusive lock"
msgstr "No s'ha pogut obtenir un bloqueig exclusiu"

#: ../DistUpgrade/DistUpgradeController.py:226
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Normalment això es deu al fet que teniu un altre gestor de paquets en "
"execució (com  ara l'apt-get o l'aptitude) i cal que abans el tanqueu."

#: ../DistUpgrade/DistUpgradeController.py:284
msgid "Upgrading over remote connection not supported"
msgstr "No és possible actualitzar a través d'una connexió remota"

#: ../DistUpgrade/DistUpgradeController.py:285
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Esteu duent a terme l'actualització a través d'una connexió SSH remota amb "
"un frontal que no admet aquesta funció. Intenteu realitzar una actualització "
"en mode text amb l'ordre «do-release-upgrade». \n"
"\n"
"S'interromprà l'actualització. Torneu-ho a intentar sense SSH."

#: ../DistUpgrade/DistUpgradeController.py:299
msgid "Continue running under SSH?"
msgstr "Voleu continuar treballant via SSH?"

#: ../DistUpgrade/DistUpgradeController.py:300
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Segons sembla, aquesta sessió s'està executant per sota de SSH. Actualment "
"no és recomanable realitzar una actualització per sota de SSH, atès que en "
"cas de fallada la recuperació és més difícil.\n"
"\n"
"Si continueu, s'iniciarà un dimoni addicional al port «%s».\n"
"Voleu continuar?"

#: ../DistUpgrade/DistUpgradeController.py:314
msgid "Starting additional sshd"
msgstr "S'està iniciant un sshd addicional"

#: ../DistUpgrade/DistUpgradeController.py:315
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Per facilitar la recuperació en cas de fallada, s'iniciarà un sshd "
"addicional al port «%s». Si alguna cosa anés malament amb l'ssh en ús, podeu "
"fer servir l'addicional.\n"

#: ../DistUpgrade/DistUpgradeController.py:323
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Si feu servir un tallafoc, necessitareu obrir temporalment aquest port. Atès "
"que això és potencialment perillós, no es fa automàticament. Per exemple, "
"podeu obrir el port amb:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeController.py:401
#: ../DistUpgrade/DistUpgradeController.py:425
msgid "Can not upgrade"
msgstr "No es pot actualitzar"

#: ../DistUpgrade/DistUpgradeController.py:402
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Aquesta eina no permet l'actualització de «%s» a «%s»."

#: ../DistUpgrade/DistUpgradeController.py:426
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:455
#, python-format
msgid "Can not write to '%s'"
msgstr "No es pot escriure a «%s»"

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"No es pot escriure al directori del sistema «%s», per la qual cosa "
"l'actualització no pot continuar.\n"
"Assegureu-vos que aquest directori tingui els permisos d'escriptura adients."

#: ../DistUpgrade/DistUpgradeController.py:467
msgid "Include latest updates from the Internet?"
msgstr "Voleu incloure les darreres actualitzacions d'Internet?"

#: ../DistUpgrade/DistUpgradeController.py:468
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"El sistema d'actualitzacions pot utilitzar Internet per baixar i instal·lar "
"automàticament les últimes actualitzacions. Si disposeu d'una connexió de "
"xarxa, aquesta opció és molt recomanable.\n"
"\n"
"L'actualització trigarà més, però en acabar tindreu un sistema completament "
"actualitzat. Podeu optar per no fer-ho, però aviat haureu d'actualitzar el "
"sistema.\n"
"Si responeu «no» la xarxa no s'utilitzarà per a res."

#: ../DistUpgrade/DistUpgradeController.py:646
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""
"No està pensat per a éssers humans mentre %s està en fase de desenvolupament"

#: ../DistUpgrade/DistUpgradeController.py:747
#, python-format
msgid "disabled on upgrade to %s"
msgstr "desactivat en actualitzar a %s"

#: ../DistUpgrade/DistUpgradeController.py:804
msgid "No valid sources.list entry found"
msgstr "No s'ha trobat cap entrada de sources.list vàlida"

#: ../DistUpgrade/DistUpgradeController.py:805
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""
"Quan s'estava analitzant la vostra informació de dipòsit no s'ha pogut "
"trobar cap entrada sobre %s.\n"
"\n"
"Una actualització pot no tenir èxit.\n"
"\n"
"Voleu continuar malgrat això?"

#: ../DistUpgrade/DistUpgradeController.py:817
msgid "No valid mirror found"
msgstr "No s'ha trobat cap rèplica vàlida"

#: ../DistUpgrade/DistUpgradeController.py:818
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"No s'han trobat dipòsits de programari per a l'actualització. Això pot "
"passar quan s'utilitza una rèplica interna o quan la informació de la "
"rèplica no es troba actualitzada.\n"
"\n"
"Voleu reescriure 'sources.list' igualment? Si seleccioneu 'Sí' "
"s'actualitzaran totes les entrades '%s' per '%s'.\n"
"Si seleccioneu 'No' l'actualització es cancel·larà."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Generate default sources?"
msgstr "Voleu generar les fonts per defecte?"

#: ../DistUpgrade/DistUpgradeController.py:839
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"No s'ha trobat cap entrada vàlida per a «%s» en analitzar el fitxer «sources."
"list».\n"
"\n"
"Voleu que s'afegeixin entrades per a «%s»? Si trieu que «No», es cancel·larà "
"l'actualització."

#: ../DistUpgrade/DistUpgradeController.py:874
#: ../DistUpgrade/DistUpgradeController.py:880
msgid "Repository information invalid"
msgstr "La informació dels dipòsits no és vàlida"

#: ../DistUpgrade/DistUpgradeController.py:875
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"En actualitzar la informació dels dipòsits s'ha obtingut un fitxer no vàlid, "
"per la qual cosa s'iniciarà el procés d'informe d'errors."

#: ../DistUpgrade/DistUpgradeController.py:881
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"L'actualització de la informació del dipòsit ha finalitzat amb un fitxer "
"corrupte. Per informar un error instal·leu apport i executeu  'apport-bug "
"ubuntu-release-upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:890
msgid "Third party sources disabled"
msgstr "S'han desactivat les fonts de tercers"

#: ../DistUpgrade/DistUpgradeController.py:891
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"S'han desactivat algunes entrades de tercers. Les podeu reactivar després de "
"l'actualització des de l'eina «software-properties» en el vostre gestor de "
"paquets."

#: ../DistUpgrade/DistUpgradeController.py:933
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paquet en estat inconsistent"
msgstr[1] "Paquets en estat inconsistent"

#: ../DistUpgrade/DistUpgradeController.py:936
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"El paquet «%s» es troba en un estat inconsistent i s'ha de tornar a "
"instal·lar, però no s'ha trobat l'arxiu per a fer-ho. Torneu-lo a instal·lar "
"manualment o suprimiu-lo del sistema."
msgstr[1] ""
"Els paquets «%s» es troben en un estat inconsistent i s'han de tornar a "
"instal·lar, però no s'ha trobat els arxius per a fer-ho. Torneu-los a "
"instal·lar manualment o suprimiu-los del sistema."

#: ../DistUpgrade/DistUpgradeController.py:999
msgid "Error during update"
msgstr "S'ha produït un error en l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1000
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"S'ha produït un error mentre s'actualitzava el vostre sistema. Normalment "
"això és a causa de problemes de xarxa. Comproveu la vostra connexió de xarxa "
"i torneu a intentar-ho."

#: ../DistUpgrade/DistUpgradeController.py:1043
#: ../DistUpgrade/DistUpgradeController.py:1048
msgid "EFI System Partition (ESP) not usable"
msgstr "La partició EFI System Partition (ESP) no és usable"

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""
"La vostra partició EFI System Partition (ESP) no està muntada a /boot/efi. "
"Assegureu-vos si us plau de que està configurada apropiadament i torneu a "
"provar."

#: ../DistUpgrade/DistUpgradeController.py:1049
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""
"No es pot escriure a la partició  EFI System Partition (ESP) a /boot/efi. Si "
"us plau munteu aquesta partició amb permisos de lectura i escriptura i "
"torneu a provar."

#: ../DistUpgrade/DistUpgradeController.py:1056
msgid "Not enough free disk space"
msgstr "No disposeu de suficient espai al disc"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1059
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""
"S'ha interromput l'actualització. A l'actualització li cal un total de "
"{str_total} de espai lliure al disc «{str_dir}». Si us plau allibereu com a "
"mínim {str_needed} de disc addicional a «{str_dir}».{str_remedy}"

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1064
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""
"Elimina els paquets temporals d'instal·lacions anteriors usant «sudo apt "
"clean»."

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""
"Podeu eliminar nuclis antics usant «sudo apt autoremove» i també podeu "
"establir COMPRES=xz a /etc/initramfs-tools/initramfs.conf per reduir la mida "
"del vostre initramfs."

#: ../DistUpgrade/DistUpgradeController.py:1071
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Buideu la paperera i elimineu paquets temporals d'instal·lacions anteriors "
"usant «sudo apt-get clean»."

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid "Reboot to clean up files in /tmp."
msgstr "Reinicia per fer neteja de fitxers a /tmp."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1113
#: ../DistUpgrade/DistUpgradeController.py:1989
#: ../DistUpgrade/DistUpgradeController.py:2039
msgid "Calculating the changes"
msgstr "S'estan calculant els canvis"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1157
msgid "Do you want to start the upgrade?"
msgstr "Voleu iniciar l'actualització?"

#: ../DistUpgrade/DistUpgradeController.py:1176
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1248
msgid "Upgrade canceled"
msgstr "S'ha cancel·lat l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1249
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Ara es cancel·larà l'actualització i es restaurarà a l'estat original del "
"sistema. Podeu continuar l'actualització més tard."

#: ../DistUpgrade/DistUpgradeController.py:1255
#: ../DistUpgrade/DistUpgradeController.py:1400
msgid "Could not download the upgrades"
msgstr "No s'han pogut baixar les actualitzacions"

#: ../DistUpgrade/DistUpgradeController.py:1256
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"S'ha interromput l'actualització. Comproveu la connexió a Internet o els "
"suports d'instal·lació i torneu-ho a provar. S'han mantingut tots els "
"fitxers baixats fins ara."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1349
#: ../DistUpgrade/DistUpgradeController.py:1388
#: ../DistUpgrade/DistUpgradeController.py:1504
msgid "Error during commit"
msgstr "S'ha produït un error durant l'enviament"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1351
#: ../DistUpgrade/DistUpgradeController.py:1390
#: ../DistUpgrade/DistUpgradeController.py:1543
msgid "Restoring original system state"
msgstr "S'està restaurant l'estat original del sistema"

#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1367
#: ../DistUpgrade/DistUpgradeController.py:1391
msgid "Could not install the upgrades"
msgstr "No s'han pogut instal·lar les actualitzacions"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1357
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"S'ha interromput l'actualització. Pot ser que el vostre sistema hagi quedat "
"en un estat inestable. Ara s'executarà un procés de recuperació (dpkg --"
"configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1362
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Informeu d'aquest error en un navegador a http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug i adjunteu els fitxers de /var/log/"
"dist-upgrade/ a l'informe d'error.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1401
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"S'ha interromput l'actualització. Verifiqueu la vostra connexió a Internet o "
"el suport d'instal·lació i torneu-ho a intentar. "

#: ../DistUpgrade/DistUpgradeController.py:1429
msgid "Searching for obsolete software"
msgstr "S'està cercant programari obsolet"

#: ../DistUpgrade/DistUpgradeController.py:1492
msgid "Remove obsolete packages?"
msgstr "Voleu suprimir els paquets obsolets?"

#: ../DistUpgrade/DistUpgradeController.py:1493
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Manté"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "_Remove"
msgstr "Sup_rimeix"

#: ../DistUpgrade/DistUpgradeController.py:1505
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"S'ha produït un error durant el procés de neteja. Vegeu el següent missatge "
"per a més informació. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "Required depends is not installed"
msgstr "Les dependències requerides no estan instal·lades"

#: ../DistUpgrade/DistUpgradeController.py:1582
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "La dependència requerida «%s» no està instal·lada. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1849
#: ../DistUpgrade/DistUpgradeController.py:1939
msgid "Checking package manager"
msgstr "S'està comprovant el gestor de paquets"

#: ../DistUpgrade/DistUpgradeController.py:1855
#: ../DistUpgrade/DistUpgradeController.py:1861
msgid "Preparing the upgrade failed"
msgstr "S'ha produït un error en la preparació de l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1856
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Ha fallat la preparació del sistema per l'actualització, per la qual cosa "
"s'iniciarà el procés d'informe d'errors."

#: ../DistUpgrade/DistUpgradeController.py:1862
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Ha fallat la preparació de l'actualització del sistema. Per informar d'un "
"error, instal·leu l'apport i executeu «apport-bug ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:1879
#: ../DistUpgrade/DistUpgradeController.py:1889
msgid "Getting upgrade prerequisites failed"
msgstr "S'ha produït un error en obtenir els prerequisits de l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1880
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"El sistema no ha pogut obtenir els requeriments per a l'actualització. "
"L'actualització s'interromprà i es retornarà el sistema al seu estat "
"original.\n"
"\n"
"També s'iniciarà el procés d'informe d'errors."

#: ../DistUpgrade/DistUpgradeController.py:1890
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"El sistema no ha pogut  descarregar els prerequisits per a l'actualització. "
"El procés d'actualització s'aturarà i es retornarà el sistema a l'estat "
"original.\n"
"\n"
"Per informar d'un error instal·leu apport i executeu 'apport-bug ubuntu-"
"release-upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:1923
msgid "Updating repository information"
msgstr "S'està actualitzant la informació dels dipòsits"

#: ../DistUpgrade/DistUpgradeController.py:1930
msgid "Failed to add the cdrom"
msgstr "No s'ha pogut afegir el CD-ROM"

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Sorry, adding the cdrom was not successful."
msgstr "No s'ha pogut afegir el CD-ROM correctament."

#: ../DistUpgrade/DistUpgradeController.py:1966
msgid "Invalid package information"
msgstr "La informació del paquet no és vàlida"

#: ../DistUpgrade/DistUpgradeController.py:1967
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"Després d'actualitzar la informació dels paquets, no s'ha pogut localitzar "
"el paquet essencial «%s». Això deu al fet que no teniu cap rèplica oficial "
"al llistat de fonts de programari o perquè hi ha una càrrega excessiva en la "
"rèplica que esteu utilitzant. Vegeu /etc/apt/sources.list per veure la "
"llista actual de fonts de programari.\n"
"En cas de rèplica amb sobrecàrrega, podeu intentar l'actualització més tard."

#: ../DistUpgrade/DistUpgradeController.py:1996
#: ../DistUpgrade/DistUpgradeController.py:2095
msgid "Fetching"
msgstr "S'està recollint"

#: ../DistUpgrade/DistUpgradeController.py:2003
#: ../DistUpgrade/DistUpgradeController.py:2024
#: ../DistUpgrade/DistUpgradeController.py:2045
#: ../DistUpgrade/DistUpgradeController.py:2099
msgid "Upgrading"
msgstr "S'està actualitzant"

#: ../DistUpgrade/DistUpgradeController.py:2005
msgid "Upgrade infeasible"
msgstr "L'actualizació no és factible"

#: ../DistUpgrade/DistUpgradeController.py:2006
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""
"No s'ha pogut completar l'actualització, hi ha hagut errors durant el procés "
"d'actualització."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2027
msgid "Upgrade incomplete"
msgstr "Actualització incompleta"

#: ../DistUpgrade/DistUpgradeController.py:2028
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""
"S'ha completat parcialment l'actualització però hi ha hagut errors durant el "
"procés d'actualització."

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2050
#: ../DistUpgrade/DistUpgradeController.py:2101
#: ../DistUpgrade/DistUpgradeController.py:2107
#: ../DistUpgrade/DistUpgradeController.py:2118
msgid "Upgrade complete"
msgstr "S'ha completat l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:2051
#: ../DistUpgrade/DistUpgradeController.py:2102
#: ../DistUpgrade/DistUpgradeController.py:2108
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"S'ha completat l'actualització, però s'han produït errors durant aquest "
"procés."

#: ../DistUpgrade/DistUpgradeController.py:2070
msgid "System upgrade is complete."
msgstr "S'ha completat l'actualització del sistema."

#: ../DistUpgrade/DistUpgradeController.py:2119
msgid "The partial upgrade was completed."
msgstr "L'actualització parcial s'ha completat."

#: ../DistUpgrade/DistUpgradeController.py:2147
msgid "Lock screen disabled"
msgstr "La pantalla de bloqueig està inactivada"

#: ../DistUpgrade/DistUpgradeController.py:2148
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr ""
"S'ha desactivat la vostra pantalla de bloqueig, i continuarà desactivada "
"fins que reinicieu."

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "No s'han trobat les notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "El servidor potser està sobrecarregat. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "No s'han pogut baixar les notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Comproveu la vostra connexió a Internet."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "autenticació del fitxer «%(file)s» amb la signatura «%(signature)s» "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:97
#, python-format
msgid "extracting '%s'"
msgstr "s'està extraient «%s»"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:118
#: ../DistUpgrade/DistUpgradeFetcherCore.py:119
msgid "Could not run the upgrade tool"
msgstr "No es pot executar l'eina d'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"El més probable és que sigui un error en l'eina d'actualització. Informeu "
"d'aquest error utilitzant l'ordre «ubuntu-bug ubuntu-release-upgrader-core» "
"en un terminal."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:197
msgid "Upgrade tool signature"
msgstr "Signatura de l'eina d'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool"
msgstr "Eina d'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
msgid "Failed to fetch"
msgstr "Ha fallat la baixada"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:239
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Ha fallat l'obtenció de l'actualització. Pot ser que hi hagi algun problema "
"a la xarxa. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Authentication failed"
msgstr "Ha fallat l'autenticació"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Ha fallat l'autenticació de l'actualització. Pot ser que hi hagi algun "
"problema a la xarxa o al servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
msgid "Failed to extract"
msgstr "Ha fallat l'extracció"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:250
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Ha fallat l'extracció de l'actualització. Pot ser que hi hagi algun problema "
"a la xarxa o al servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
msgid "Verification failed"
msgstr "Ha fallat la verificació"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:256
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Ha fallat la verificació de l'actualització. Pot ser que hi hagi algun "
"problema a la xarxa o al servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
msgid "Can not run the upgrade"
msgstr "No es pot executar l'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Normalment això està provocat per un sistema on /tmp s'ha muntat com a no "
"executable. Torneu-lo a muntar sense l'opció de no executable i torneu a "
"executar l'actualització."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
#, python-format
msgid "The error message is '%s'."
msgstr "El missatge d'error és «%s»."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Actualitza"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "&Cancel·la"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Actualització"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "S'estan baixant els fitxers de paquet addicionals..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Fitxer %s de %s a %s B/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Fitxer %s de %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Inseriu «%s» a la unitat «%s»"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Canvi de suport"

#: ../DistUpgrade/DistUpgradeQuirks.py:227
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"Pot ser que el vostre maquinari de gràfics no funcioni bé amb l'Ubuntu 14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:229
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""
"L'entorn d'escriptori «Unity» no és totalment compatible amb el vostre "
"maquinari gràfic. Pot ser que l'escriptori s'executi molt lentament després "
"de l'actualització. El nostre consell és que de moment mantingueu la versió "
"LTS. Per obtenir més informació vegeu https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D. Tot i això, voleu continuar amb "
"l'actualització?"

#: ../DistUpgrade/DistUpgradeQuirks.py:253
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""
"Sembla que el vostre maquinari gràfic no és totalment compatible amb "
"l'Ubuntu 12.04 LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:255
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"La compatibilitat a l'Ubuntu 12.04 LTS per al vostre maquinari gràfic Intel "
"és limitat i podríeu tenir problemes després d'actualitzar. Per obtenir més "
"informació visiteu https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForI8xx. Voleu continuar amb l'actualització?"

#: ../DistUpgrade/DistUpgradeQuirks.py:275
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"L'actualització pot reduir els efectes d'escriptori i el rendiment en jocs i "
"altres programes que facin un ús exhaustiu de processament gràfic."

#: ../DistUpgrade/DistUpgradeQuirks.py:279
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Aquest ordinador utilitza el controlador gràfic «fglrx» d'AMD. No hi ha cap "
"versió disponible d'aquest controlador que funcioni amb el vostre maquinari "
"a l'Ubuntu 10.04 LTS.\n"
"\n"
"Voleu continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:318
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:319
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:334
msgid "No ARMv6 CPU"
msgstr "No és una CPU ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:335
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"El vostre sistema utilitza una CPU ARM que és més antiga que l'arquitectura "
"ARMv6. Tots els paquets del Karmic s'han creat amb optimitzacions que "
"requereixen l'ARMv6 com a arquitectura mínima. No és possible actualitzar el "
"vostre sistema a una versió nova de l'Ubuntu amb aquest maquinari."

#: ../DistUpgrade/DistUpgradeQuirks.py:355
msgid "No init available"
msgstr "No hi ha cap sistema d'inicialització disponible"

#: ../DistUpgrade/DistUpgradeQuirks.py:356
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Segons sembla, el vostre sistema és un entorn virtual sense un dimoni "
"d'inicialització «init», com ara un servidor virtual de Linux (Linux-"
"VServer). L'Ubuntu 10.04 no pot funcionar en aquest tipus d'entorn - abans "
"cal fer una actualització de la configuració de la màquina virtual.\n"
"\n"
"Esteu segur que voleu continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:469
#, fuzzy
msgid "Connection to the Snap Store failed"
msgstr "Ha fallat la connexió a la Botiga Snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:470
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:485
msgid "Connection to Snap Store failed"
msgstr "Ha fallat la connexió a la Botiga Snap"

#: ../DistUpgrade/DistUpgradeQuirks.py:486
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"El vostre sistema no té una connexió a la Botiga Snap. Per a una experiència "
"òptima d'actualització assegureu-vos que el vostre sistema es pot connectar "
"a api.snapcraft.io.\n"
"Encara voleu continuar amb l'actualització?"

#: ../DistUpgrade/DistUpgradeQuirks.py:495
msgid "Outdated snapd package"
msgstr "Paquet snapd desactualitzat"

#: ../DistUpgrade/DistUpgradeQuirks.py:496
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"El vostre sistema no te l'última versió de snapd. Si us plau actualitzeu la "
"versió de snapd al vostre sistema per millorar l'experiència "
"d'actualització.\n"
"Encara voleu continuar amb l'actualització?"

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:519
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:550
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:556
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:560
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:563
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:592
msgid "PAE not enabled"
msgstr "El PAE no està habilitat"

#: ../DistUpgrade/DistUpgradeQuirks.py:593
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"El sistema incorpora una CPU que no té el PAE habilitat. L'Ubuntu funciona "
"amb sistemes sense PAE només fins a la versió 12.04. Per actualitzar a una "
"versió posterior de l'Ubuntu, haureu d'habilitar el PAE (si és possible). "
"Consulteu la documentació següent per obtenir més informació:\n"
"http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:916
msgid "Checking for installed snaps"
msgstr "S'està comprovant si hi ha snaps instal·lats"

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Utilitza el camí especificat per cercar un CDROM amb paquets actualitzables"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Utilitzeu un frontal. Actualment es disposa de: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr "*OBSOLET* aquesta opció s'ignorarà"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Realitza només una actualització parcial (sense reescriure el sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "Inhabilita la compatibilitat amb el GNU screen"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "Estableix el «datadir» (directori de dades)"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr "Actualitza a la versió de desenvolupament"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "S'ha completat el recull"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "S'està obtenint el fitxer %li de %li a %s B/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Queden %s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "S'està obtenint el fitxer %li de %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "S'estan aplicant els canvis"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "problemes de dependències - es deixarà sense configurar"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "No s'ha pogut instal·lar «%s»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"L'actualització continuarà, però pot ser que el paquet «%s» no sigui "
"funcional. Hauríeu de considerar l'enviament d'un informe d'error sobre "
"aquest fet."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Voleu reemplaçar el fitxer de\n"
"configuració personalitzat «%s»?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Perdreu tots els canvis realitzats en el fitxer de configuració si trieu "
"reemplaçar-lo per una versió més nova."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "No s'ha trobat l'ordre «diff»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "S'ha produït un error greu"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informeu d'això com un error (si no és que ja ho heu fet) i adjunteu els "
"fitxers /var/log/dist-upgrade/main.log i /var/log/dist-upgrade/apt.log al "
"vostre informe. S'ha interromput l'actualització.\n"
"S'ha desat el vostre fitxer sources.list original a /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "S'ha premut Ctrl+C"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Això interromprà l'operació, amb la qual cosa pot ser que es malmeti el "
"sistema. Esteu segur que ho voleu fer?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Per a evitar una possible pèrdua de dades, tanqueu tots els documents i "
"aplicacions."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Ja no mantinguts per Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Es desactualitzaran (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "Se suprimiran (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "Ja no són necessaris (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "S'instal·laran (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "S'actualitzaran (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Mostra les diferències >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Oculta les diferències"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "S'ha produït un error"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "&Tanca"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "Mostra el terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< Oculta el terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "Informació"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "&Inicia l'actualització"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Detalls"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "Ja no es mantenen %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "Suprimeix %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Suprimeix %s (s'havia instal·lat automàticament)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "Instal·la %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "Actualitza %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Cal que reinicieu el sistema"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Reinicieu el sistema per a completar l'actualització</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Reinicia"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Voleu cancel·lar l'actualització en curs?</big></b>\n"
"\n"
"El sistema pot quedar inusable si la cancel·leu. És molt recomanable "
"continuar amb l'actualització."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "Voleu cancel·lar l'actualització?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dia"
msgstr[1] "%li dies"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li hora"
msgstr[1] "%li hores"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minut"
msgstr[1] "%li minuts"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li segon"
msgstr[1] "%li segons"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"La baixada trigarà aproximadament %s amb una connexió DSL d'1 Mbit, i "
"aproximadament %s amb un mòdem de 56 k."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "La baixada trigarà aproximadament %s amb la vostra connexió. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "S'està preparant l'actualització"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "S'estan obtenint canals de programari nous"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "S'estan obtenint els paquets nous"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "S'estan instal·lant les actualitzacions"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "S'està netejant"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d paquet instal·lat ja no està mantingut per Canonical. Així i tot, "
"encara podeu obtenir assistència de la comunitat."
msgstr[1] ""
"%(amount)d dels paquets instal·lats ja no estan mantinguts per Canonical. "
"Així i tot, encara podeu obtenir assistència de la comunitat."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "Se suprimirà %d paquet."
msgstr[1] "Se suprimiran %d paquets."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "S'instal·larà %d paquet nou."
msgstr[1] "S'instal·laran %d paquets nous."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "S'actualitzarà %d paquet"
msgstr[1] "S'actualitzaran %d paquets"

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Heu de baixar un total de %s. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"La instal·lació de l'actualització pot trigar unes quantes hores. Un cop "
"hagi finalitzat la baixada dels fitxers, el procés no es pot cancel·lar."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"La baixada i instal·lació de l'actualització pot trigar unes quantes hores. "
"Un cop hagi finalitzat la baixada dels fitxers, el procés no es pot "
"cancel·lar."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "La supressió de paquets pot trigar unes quantes hores. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "El programari d'aquest ordinador està actualitzat."

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"No hi ha actualitzacions disponibles per al vostre sistema. Ara es "
"cancel·larà l'actualització."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Cal que reinicieu el sistema"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"L'actualització ha finalitzat i cal reiniciar el sistema. Voleu fer-ho ara?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Progrés: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informeu d'això com un error i adjunteu els fitxers /var/log/dist-upgrade/"
"main.log i /var/log/dist-upgrade/apt.log al vostre informe. S'ha interromput "
"l'actualització.\n"
"S'ha desat el vostre fitxer sources.list original a /etc/apt/sources.list."
"distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "S'està interrompent"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Degradat:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Premeu la tecla de retorn per continuar"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Continua [sN] "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Detalls [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "s"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ja no es mantenen: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Suprimeix: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Instal·la: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Actualitza: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Continua [Sn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Cal reiniciar el sistema per a finalitzar l'actualització.\n"
"Si seleccioneu «s», es reiniciarà el sistema."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "S'està baixant el fitxer %(current)li de %(total)li a %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "S'està baixant el fitxer %(current)li de %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Mostra el progrés dels fitxers individuals"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Cancel·la l'actualització"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Reprèn l'actualització"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Voleu cancel·lar l'actualització?</big></b>\n"
"\n"
"Si ho feu, pot ser que el sistema sigui inutilitzable. És extremament "
"recomanable continuar amb l'actualització."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Comença l'actualització"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Reemplaça"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Diferències entre els fitxers"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Informa de l'error"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Continua"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Voleu iniciar l'actualització?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Reinicieu el sistema per a completar l'actualització</big></b>\n"
"\n"
"Deseu la vostra feina abans de continuar."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Actualització de la distribució"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr "<b><big>Actualitzant Ubuntu a la versió %s</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "S'estan configurant els canals de programari nous"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "S'està reiniciant l'ordinador"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Terminal"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Actualitza"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Hi ha disponible una versió nova de l'Ubuntu. Voleu actualitzar el vostre "
"sistema?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "No actualitzis"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Demana-m'ho més tard"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Actualitza"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Heu triat no actualitzar a la versió nova de l'Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Podeu actualitzar més tard obrint el gestor d'actualitzacions i fent clic a "
"«Actualitza»."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Realitza una actualització de la versió"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Cal que us autentiqueu per actualitzar l'Ubuntu."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Realitza una actualització parcial"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Cal que us autentiqueu per fer una actualització parcial."

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Mostra la versió i surt"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "El directori que conté els fitxers de dades"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Executa el frontal especificat"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "S'està executant una actualització parcial"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "S'està baixant l'eina d'actualització a una versió nova"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""
"Si s'està usant la versió amb suport més recent, actualitzeu a la versió de "
"desenvolupament"

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Proveu d'actualitzar a l'última versió fent servir l'actualitzador de "
"$distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Executeu un mode d'actualització especial.\n"
"Actualment disposeu del mode «desktop» per a actualitzacions de sistemes "
"d'escriptori i del mode «server» per a sistemes servidors."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Només verifica si hi ha disponible una versió nova i informa del resultat "
"mitjançant el codi de sortida"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr "Les opcions --devel-release i --proposed són"

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr "mútuament excloents. Feu servir només una d'elles."

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "S'està cercant una versió nova de l'Ubuntu"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "La vostra versió de l'Ubuntu ja no està mantinguda."

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Per obtenir més informació sobre l'actualització, aneu a:\n"
"%(url)s\n"

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr "Les actualitzacions a la versió de desenvolupament sols estan "

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr "disponibles per a la versió suportada més recent."

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr "No s'ha trobat cap nova versió."

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr "En aquests moments no es pot efectuar l'actualització de versió"

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"En aquests moments no es pot efectuar l'actualització de versió. Torneu-ho a "
"provar més tard. El servidor ha respost: «%s»"

#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "La versió nova «%s» ja està disponible."

#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Executeu «do-release-upgrade» per actualitzar a la nova versió."

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr ""

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr "Gestor d'actualitzacions"

#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Hi ha disponible una actualització de l'Ubuntu %(version)s"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Heu triat no actualitzar a l'Ubuntu %s"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Comprova si és possible actualitzar a la darrera versió de desenvolupament"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr "Afegeix informació de depuració"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "S'ha produït un problema irresoluble mentre es calculava "
#~ "l'actualització:\n"
#~ "%s\n"
#~ "\n"
#~ "Això pot ser degut a:\n"
#~ " * l'actualització a una versió en desenvolupament de l'Ubuntu\n"
#~ " * l'execució de l'actual versió en desenvolupament de l'Ubuntu\n"
#~ " * la instal·lació de paquets no oficials de programari i no "
#~ "proporcionats per l'Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "No teniu instal·lats el paquet ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop o edubuntu-desktop i no s'ha pogut detectar la versió de l'Ubuntu "
#~ "que esteu utilitzant.\n"
#~ "Instal·leu un d'aquests paquets abans de continuar; per fer-ho podeu "
#~ "utilitzar el Synaptic o l'apt-get"

#~ msgid "Sandbox setup failed"
#~ msgstr "Ha fallat la configuració d'un entorn de proves"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "No ha estat possible crear un entorn de proves."

#~ msgid "Sandbox mode"
#~ msgstr "Mode de prova"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "Aquesta actualització s'està executant en mode de proves. Tots els canvis "
#~ "s'escriuran a «%s» i es perdran en reiniciar.\n"
#~ "\n"
#~ "Cap dels canvis fets als directoris del sistema seran permanents."

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "La vostra instal·lació del Python està malmesa. Reviseu l'enllaç simbòlic "
#~ "«/usr/bin/python»."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "El paquet «debsig-verify» està instal·lat"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "L'actualització no pot continuar amb aquest paquet instal·lat.\n"
#~ "Elimineu-lo amb el Synaptic o amb l'ordre «apt-get remove debsig-verify» "
#~ "i torneu a executar l'actualització."

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "No s'ha trobat cap entrada de rèplica per a l'actualització a la vostra "
#~ "informació sobre els dipòsits. Això pot ésser degut a què estigueu "
#~ "executant una rèplica interna o que les dades de les rèpliques no "
#~ "estiguin actualitzades.\n"
#~ "\n"
#~ "Voleu sobreescriure el fitxer «sources.list» de totes maneres? Si trieu "
#~ "que «Sí», totes les entrades «%s» s'actualitzaran a «%s». Si trieu que "
#~ "«No», es cancel·larà l'actualització."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "S'ha interromput l'actualització. Cal un total de %s d'espai lliure al "
#~ "disc «%s», per la qual cosa hauríeu d'alliberar un mínim de %s d'espai "
#~ "addicional al disc «%s». Buideu la paperera i suprimiu els paquets "
#~ "temporals d'anteriors instal·lacions utilitzant l'ordre «sudo apt-get "
#~ "clean»."

#~ msgid "evms in use"
#~ msgstr "S'està utilitzant l'evms"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "El vostre sistema utilitza el gestor de volum «evms» a /proc/mounts. "
#~ "Aquest programari no es mantindrà més, desconnecteu-lo i torneu a "
#~ "executar l'actualització."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Aquest ordinador utilitza el controlador gràfic «nvidia» d'NVIDIA. No hi "
#~ "ha cap versió disponible d'aquest controlador que funcioni amb el vostre "
#~ "maquinari a l'Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Voleu continuar?"

#~ msgid "No i686 CPU"
#~ msgstr "Sense CPU i686"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "El vostre sistema utilitza una CPU i586 o una CPU que no té l'extensió "
#~ "«cmov». Tots els paquets s'han construït amb optimitzacions que "
#~ "requereixen un i686 com a arquitectura mínima. No és possible actualitzar "
#~ "el sistema a una versió nova de l'Ubuntu amb aquest maquinari."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Actualització en entorn de proves utilitzant aufs"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "Prova d'actualitzar en un entorn de proves amb aufs"

#~ msgid "No new release found"
#~ msgstr "No s'ha trobat cap versió nova"

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "Cal autenticació per realitzar una actualització de la versió"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr "Cal autenticació per realitzar una actualització parcial"

#~ msgid ""
#~ "If none of this applies, then please report this bug using the command "
#~ "'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
#~ msgstr ""
#~ "Si res d'això és aplicable, informeu d'aquest error utilitzant l'ordre "
#~ "«ubuntu-bug ubuntu-release-upgrader-core» en un terminal."

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "S'ha produït un problema irresoluble mentre es feia el càlcul per a "
#~ "l'actualització.\n"
#~ "\n"
#~ " Això pot ser degut a:\n"
#~ " * l'actualització a una versió en desenvolupament de l'Ubuntu\n"
#~ " * l'execució de l'actual versió en desenvolupament de l'Ubuntu\n"
#~ " * la instal·lació de paquets no oficials de programari no proveïts per "
#~ "l'Ubuntu\n"
#~ "\n"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.04</big></b>"
#~ msgstr "<b><big>Actualitzant Ubuntu a la versió 15.04</big></b>"

#~ msgid "apt architecture is i386"
#~ msgstr "L'arquitectura de l'apt és i386"

#~ msgid ""
#~ "Upgrades from 18.04 on the i386 architecture are not supported at this "
#~ "time."
#~ msgstr ""
#~ "Les actualitzacions des de 18.04 sobre l'arquitectura i381 no estan "
#~ "suportades en aquest moment."
